packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -1478,6 +1478,56 @@ class PythonModule implements PythonType {
     code.line('publication.publish()');
   }
 
+  /**
+   * Emit the bin scripts if bin section defined.
+   */
+  public emitBinScripts(code: CodeMaker): string[] {
+    const scripts: string[] = [];
+    if (this.loadAssembly) {
+      if (this.assembly.bin != null) {
+        for (const [name, script_path] of Object.entries(this.assembly.bin)) {
+          const script_file = path.join(
+            'src',
+            pythonModuleNameToFilename(this.pythonName),
+            'bin',
+            name,
+          );
+          code.openFile(script_file);
+          code.line('#!/usr/bin/env python');
+          code.line();
+          code.line('import jsii');
+          code.line('import sys');
+          code.line();
+          emitList(
+            code,
+            '__jsii_assembly__ = jsii.JSIIAssembly.load(',
+            [
+              JSON.stringify(this.assembly.name),
+              JSON.stringify(this.assembly.version),
+              JSON.stringify(this.pythonName.replace('._jsii', '')),
+              `${JSON.stringify(this.assemblyFilename)}`,
+            ],
```

Comment:
You will probably run into issues doing that, because this flow never loads the current package's dependencies into the kernel...

Instead of re-doing this, you should import the module that actually initializes things correctly:
```py
import ._jsii # <- And voilà, dependencies and the current module are loaded!
```

Reply:
Do not really understand what do you mean here?